src/components/date-picker/index.js

```diff
@@ -0,0 +1,2 @@
+import DatePicker from './date-picker';
+export default DatePicker;
```

Comment:
In `index.js` files I prefer to use 
```
export {
  DatePicker,
};
```

It's easier to extend with additional exports later.